src/frontend/src/app/core/ui/info-icon/info-icon.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-info-icon',
+  standalone: true,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      stroke-width="1.5"
+      stroke="currentColor"
+      class="w-5 h-5 ml-2"
+    >
+      <path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        d="m11.25 11.25.041-.02a.75.75 0 0 1 1.063.852l-.708 2.836a.75.75 0 0 0 1.063.853l.041-.021M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Zm-9-3.75h.008v.008H12V8.25Z"
+      ></path>
+      <title>{{ infoText }}</title>
+    </svg>
+  `,
+})
+export class InfoIconComponent {
+  @Input()
```

Comment:
Sorry, I missed this while we reviewed it together. It's generally not safe to use the non-null operator, but especially so for `@Input` properties because we cannot guarantee how the component will be used.

We can get Angular to help us though, by using the [`required`](https://angular.dev/api/core/Input) property on the input's metadata.

```suggestion
  @Input({ required: true })
```